tutorials/nlp/Token_Classification-BioMegatron.ipynb

```diff
@@ -0,0 +1,602 @@
+   "metadata": {},
+   "source": [
+    "For our purposes, we will consider any identified category (such as \"Modifier\", \"Specific Disease\", and a few others) to generally be a \"disease\".\n",
+    "\n",
+    "Let's download the dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATA_DIR = \"DATA_DIR\"\n",
+    "os.makedirs(DATA_DIR, exist_ok=True)\n",
+    "os.makedirs(os.path.join(DATA_DIR, 'NER'), exist_ok=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('Downloading NCBI data...')\n",
+    "wget.download('https://www.ncbi.nlm.nih.gov/CBBresearch/Dogan/DISEASE/NCBI_corpus.zip', DATA_DIR)\n",
+    "! unzip {DATA_DIR}/NCBI_corpus.zip -d {DATA_DIR}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you want to see more examples, you can explore the text of the corpus using the file browser to the left, or open files directly:\n",
+    "\n",
+    "* [DATA_DIR/NCBI_corpus_training.txt](DATA_DIR/NCBI_corpus_testing.txt)\n",
+    "* [DATA_DIR/NCBI_corpus_testing.txt](DATA_DIR/NCBI_corpus_testing.txt)\n",
+    "* [DATA_DIR/NCBI_corpus_development.txt](DATA_DIR/NCBI_corpus_development.txt)\n",
+    "\n",
+    "We have two datasets derived from this corpus:  a text classification dataset and a named entity recognition (NER) dataset.  The text classification dataset labels the abstracts among three broad disease groupings.  We'll use this simple split to demonstrate the NLP text classification task.   The NER dataset labels individual words as diseases.  This dataset will be used for the NLP NER task.  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pre-process dataset\n",
+    "A pre-processed NCBI-disease dataset for NER can be found [here](https://github.com/spyysalo/ncbi-disease/tree/master/conll) or [here](https://github.com/dmis-lab/biobert#datasets).<br>\n",
+    "We download the files under {DATA_DIR/NER} directory."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wget.download('https://raw.githubusercontent.com/spyysalo/ncbi-disease/master/conll/train.tsv', os.path.join(DATA_DIR, 'NER'))\n",
+    "wget.download('https://raw.githubusercontent.com/spyysalo/ncbi-disease/master/conll/devel.tsv', os.path.join(DATA_DIR, 'NER'))\n",
+    "wget.download('https://raw.githubusercontent.com/spyysalo/ncbi-disease/master/conll/test.tsv', os.path.join(DATA_DIR, 'NER'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "NER_DATA_DIR = 'DATA_DIR/NER'\n",
+    "!ls -lh $NER_DATA_DIR"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Convert these to NeMo compatible format:"
+   ]
```

Comment:
Please a few more words describing what we're doing + link to the NER notebook for more details on the format